crates/nu-parser/src/parser.rs

```diff
@@ -5262,6 +5297,17 @@ pub fn parse_record(
     let mut start = span.start;
     let mut end = span.end;
 
+    if let false = balanced_pair_between_qoutes(b'{', b'}', bytes) {
```

Comment:
I think we can use `if not balanced_pair_between_qoutes(b'{', b'}', bytes)` simply.

Reply:
Did you mix `Nu` and `Rust` syntax :)